doctor/cores/text_processor.py

```
@@ -0,0 +1,462 @@
+    processed_text: str
+    entities: List[MedicalEntity]
+    keywords: List[str]
+    sentiment: str
+    language: str
+    medical_relevance: float
+
+
+class DoctorTextProcessorCore:
+    """
+    هسته پردازش متن برای اپ Doctor
+    مسئول پردازش نوت‌های پزشکی، استخراج اطلاعات و تولید گزارش‌ها
```

Ruff (0.12.2)

Docstring contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)